README.md

```diff
@@ -55,6 +55,7 @@ Lassie can be controlled with a config file. To enable lassie for a project you
 
 ```json5
 {
+  "$schema": "https://geprog.github.io/lassie-bot-dog/lassie.schema.json",
```

Comment:
@lukashass Seems to me that the schema is release under: `https://geprog.github.io/lassie-bot-dog/config/lassie.schema.json`

Reply:
@anbraten No. Have a look, its already there :wink: https://geprog.github.io/lassie-bot-dog/lassie.schema.json